voluncheer/templates/profiles/posttogallery.html

```diff
@@ -27,15 +27,11 @@
             <a href="{% url 'home' %}">PROFILE</a>
             <hr align="center">
         </div>
-        <div class="url">
-            <a class="not-implemented-yet" href="#">SETTINGS</a>
-            <hr align="center">
-        </div>
         <div class="url">
             <a href="{% url 'saved_events' %}" >MY VOLUNCHEERS</a>
             <hr align="center">
         </div>
-        {% if user.pk is not curr_user.pk %}
+        {% if user.pk is not volunteer.pk %}
```

Comment:
Why are you changing curr_user to volunteer? I think curr_user is for checking if one is viewing their own profile. No need to differentiate between volunteer and organization.